migrations/20220913092444_convert_votes_to_reactions.js

```diff
@@ -0,0 +1,65 @@
+
+exports.up = async function (knex) {
+  await knex.raw('alter table votes drop constraint if exists uq_vote_1')
+  await knex.schema.renameTable('votes', 'reactions')
+
+  await knex.schema.table('reactions', table => {
+    table.text('emoji_base')
+    table.text('emoji_full')
+    table.text('emoji_label')
+    table.text('entity_type')
+    table.renameColumn('post_id', 'entity_id')
+    table.renameColumn('date_voted', 'date_reacted')
+    table.index('emoji_base', 'idx_reactions_emoji_full')
+    table.index('entity_type', 'idx_reactions_entity_type')
+    table.index('entity_id', 'idx_reactions_entity_id')
+  })
+
+  await knex.raw('DROP INDEX ix_vote_post_14')
+
+  // migrate prior data
+  // 'U+1F44D' but encoded in UTF-16 (because javascript) => '\uD83D\uDC4D', thumbs up emoji, :thumbs up:
+  await knex('reactions').update({
+    emoji_base: '\uD83D\uDC4D',
+    emoji_full: '\uD83D\uDC4D',
+    entity_type: 'post',
+    emoji_label: ':thumbs up:'
+  })
+
+  await knex.schema.table('posts', table => {
+    table.jsonb('reactions_summary')
+    table.renameColumn('num_votes', 'num_people_reacts')
```

Comment:
so wait, is this the total number of reactions, or the total number of people who have reacted? could be either num_reactions or num_people_reacted